packages/applications-service-api/prisma/schema.prisma

```diff
@@ -86,8 +88,10 @@ model ServiceUser {
   organisationName String?
   role             String?
   serviceUserType  String?
-  createdAt        DateTime? @default(now())
-  modifiedAt       DateTime? @default(now())
+  representative   Representation[] @relation("representative")
```

Comment:
Think this is going to give us key constraint issues, don't think we want these?

Reply:
I thought I had reverted this! Thanks for catching this.